src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/AutomationElementExtensions.cs

```diff
@@ -310,7 +309,6 @@ private static void ThrowUnableToFindChildException(string path, IUIAutomationEl
     /// support the <see cref="IUIAutomationTogglePattern"/>.
     /// </summary>
     /// <param name="element"></param>
```

Comment:
Not going to continue commenting on these, but there's a bunch :slightly_smiling_face:

Reply:
StyleCop Analyzers has analyzers for these cases if we decide they matter.